src/Controller/AbstractController.php

```diff
@@ -24,5 +24,10 @@ public function __construct()
             ]
         );
         $this->twig->addExtension(new DebugExtension());
+
+        //add user id to global twig variable
+        if (!empty($_SESSION['user_id'])) {
+            $this->twig->addGlobal('user_id', $_SESSION['user_id']);
```

Comment:
on y est presque ;-)
là maintenant l'intéret du addGlobal est de passer toutes les infos du user à Twig, pour que tu puisses facilement récupérer ses infos (nom, email...) et les exploiter si besoin
donc tu vas plutot passer             $this->twig->addGlobal('user_id', $user); en ayant recup $user = $userManager->selectOneById('$_SESSION['user_id']